docker/images/aws_generate_docker_image_policy_file_v1.0.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Check whether the required CICD_SCRIPTS_AWS_ACCOUNT_IDS is set
+: ${CICD_SCRIPTS_AWS_ACCOUNT_IDS?'CICD_SCRIPTS_AWS_ACCOUNT_IDS is required for this script to function'}
+
+# Record directory of script for convenience
+SCRIPT_PATH=$(readlink -f "${0}")
+SCRIPT_DIR=$(dirname "${SCRIPT_PATH}")
+
+DOCKER_IMAGE_POLICY_FILE="${SCRIPT_DIR}"/aws_generate_docker_image_policy_file.json
+cat << EOF > "${DOCKER_IMAGE_POLICY_FILE}"
+{
+  "Version": "2008-10-17",
```

Comment:
2008-10-17 is an old version of the policy language. We should not use this version for any new policies or when existing policies.

```suggestion
  "Version": "2012-10-17",
```